src/Tribe/Tickets.php

```diff
@@ -1389,5 +1393,15 @@ public static function get_login_url() {
 			 */
 			return apply_filters( 'tribe_tickets_ticket_login_url', $login_url );
 		}
+
+		/**
+		 * @param $return
```

Comment:
if the check-in or uncheck-in failed then there is no use in re-updating the cache, `return` will be `true` or `false` to reflect that